template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/RazorPagesWeb-CSharp/Pages/Account/Manage/_Layout.cshtml

```diff
@@ -9,7 +9,7 @@
     <hr />
     <div class="row">
         <div class="col-md-3">
-            @await Html.PartialAsync("ManageNav")
+            @await Html.PartialAsync("_ManageNav")
```

Comment:
Should all partials following this naming pattern? If yes, then we should rename StatusMessage.cshtml as well